pkg/detectors/ringcentral/ringcentral.go

```diff
@@ -20,10 +20,11 @@ type Scanner struct {
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
-	client = common.SaneHttpClient()
+	client = detectors.DetectorHttpClientWithNoLocalAddresses
```

Comment:
Only odd thing I'm seeing is `ringcentral` failing with the changes because verification is failing. I'm guessing that is because of the `NoLocalAddress` change.

Reply:
The verification isn't against a local address though, so I'm not sure.
https://github.com/trufflesecurity/trufflehog/blob/d125e64751edc2e999bb0c32e4230debe42f0af2/pkg/detectors/ringcentral/ringcentral.go#L61

Reply:
Reverted this one. It might need to follow redirects.